frontend/utils/events.js

```diff
@@ -6,9 +6,7 @@ export const REPLACED_STATE = 'replaced-state'
 export const CONTRACTS_MODIFIED = 'contracts-modified'
 
 export const OPEN_MODAL = 'open-modal'
-export const LOAD_MODAL = 'load-modal'
 export const CLOSE_MODAL = 'close-modal'
-export const UNLOAD_MODAL = 'unload-modal'
```

Comment:
💯%

DRY. Removing redundancies.

This is how you know you're doing the right thing. Chipping away the cruft and revealing the beauty hidden within the code. 👍🥇🏆